internal/analytics/deferred.go

```diff
@@ -21,6 +21,18 @@ type deferredData struct {
 
 const deferredCfgKey = "deferrer_analytics"
 
+func SetDeferred(da bool) {
```

Comment:
@daved ran into this while addressing this story so added it here; `sendDeferred` was being called for commands where defer=true.